requirements.txt

```diff
@@ -10,5 +10,4 @@ boto3>=1.4.0
 moto==1.3.7
```

Comment:
[Python 3.5 is support-only](https://www.python.org/downloads/release/python-356/), maybe we can drop it as well and be ≥3.6? Looks like we could update a lot of version pins if so

Reply:
(`six` likewise)